main.scss

```diff
@@ -5,174 +5,194 @@
 @import "o-grid/main";
 @import "o-hoverable/main";
 
-$o-ft-footer-spacing-unit: 20px;
+/// Silent mode
+/// @type Bool
+$o-footer-is-silent: true !default;
 
-@include oFontsInclude(BentonSans);
+$o-footer-spacing-unit: 20px;
 
 // TODO: Remove this when a default shared brand color is defined in o-colors
-@include oColorsSetUseCase(o-ft-footer-brand, border, claret);
-
-@include oColorsSetUseCase(o-ft-footer, background, grey-tint5);
-@include oColorsSetUseCase(o-ft-footer, text, white);
-@include oColorsSetUseCase(o-ft-footer__pearson, background, grey-tint4);
-@include oColorsSetUseCase(o-ft-footer__divider, border, white);
-@include oColorsSetUseCase(o-ft-footer-item, text, white);
-
-.o-ft-footer {
-	@include oColorsFor(o-ft-footer, text background);
-	@include oFtTypographyFontSize(14);
-
-	margin-top: 40px;
-	padding: 0;
-	border-top: 5px solid oColorsGetColorFor(o-ft-footer-brand brand, border);
-	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
-	line-height: $o-ft-footer-spacing-unit;
-
-	> nav {
-		margin-top: $o-ft-footer-spacing-unit;
-		margin-bottom: $o-ft-footer-spacing-unit;
-		padding-bottom: $o-ft-footer-spacing-unit;
-		border-bottom: oColorsGetColorFor(o-ft-footer__divider, border) dotted 1px;
-	}
+@include oColorsSetUseCase(o-footer-brand, border, claret);
+
+@include oColorsSetUseCase(o-footer, background, grey-tint5);
+@include oColorsSetUseCase(o-footer, text, white);
+@include oColorsSetUseCase(o-footer__pearson, background, grey-tint4);
+@include oColorsSetUseCase(o-footer__divider, border, white);
+@include oColorsSetUseCase(o-footer-item, text, white);
+
+@if $o-footer-is-silent == false {
+	@include oFontsInclude(BentonSans);
 
-	ul {
-		list-style-type: none;
+	.o-footer {
```

Comment:
> If there are no mixins, it can't really be used outside in silent mode anyway

Yep. Apparently there is a case for being able to turn (at least part of) the output off. Maybe Silent mode isn't the right way to do this.

See https://github.com/Financial-Times/o-ft-footer/issues/22 - will discuss further in that issue.

> Can o-footer colours be overwritten after o-footer has set them? I ask because I wonder if there's a use case for the footer but in a different colour. Header is a nice neutral grey but the footer, being a shade of pink, may clash on some microsites?

I believe color use cases are supposed to fill that need, no?
If not, can you open an issue with a real world use case so that we can assess what the best way to do this is?